specification/computeschedule/ComputeSchedule.Management/client.tsp

```diff
@@ -79,3 +79,15 @@ using Microsoft.ComputeSchedule;
   "csharp"
 );
 @@clientName(StartResourceOperationResponse.type, "ResourceType", "csharp");
+
+@@clientName(SubmitDeallocateRequest.correlationid, "correlationId", "csharp");
```

Comment:
should we also change this to `location`?
https://github.com/ArthurMa1978/azure-rest-api-specs/blob/c6c755d911c243de456bfcbb2ddc6f98ea731e13/specification/computeschedule/ComputeSchedule.Management/models.tsp#L103

Reply:
Is this comment still valild?

Reply:
No, service team may change this in next version